schema/tasks.py

```diff
@@ -0,0 +1,28 @@
+from pydantic import BaseModel, EmailStr, Field,field_validator
+from typing import Optional
+from datetime import datetime
+from enum import Enum
+class Status(str,Enum):
+    todo = "todo"
+    in_progress = "in_progress"
+    done = "done"
+
```

Comment:
You can use inheritance in OOP to add more models to reduce validate outside models.
Example:
`class BaseUser(BaseModel):
user_name: str,
email: str
#function validate for user_name out here

class UserRegister(BaseUser):
password: str`
#all attributes and validation for attribute of BaseUser will include in this class